instagram/src/Components/PostContainer.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import CommentSection from "./CommentSection.js";
+import SearchBarHeader from "./Search";
+
+// import dummyData
+import dummyData from "./dummy-data";
+
+class PostContainer extends Component {
+
+    constructor(props) {
+        super(props);
+
```

Comment:
time saving suggestion - in modern and current React you can usually skip the constructor and super calls and just open with this...
```suggestion
state ={}
```

Reply:
ooo thanks, i'll check it out!